packages/services/api/src/modules/policy/resolvers.ts

```diff
@@ -0,0 +1,153 @@
+        organizationPolicy: orgLevel,
+        projectPolicy: projectLevel,
+      };
+    },
+  },
+  SchemaPolicyRule: {
+    id: r => r.name,
+    description: r => r.description,
+    configJsonSchema: r => r.schema,
+    recommended: r => r.recommended,
+    documentationUrl: r => r.url ?? null,
+  },
+  Query: {
+    schemaPolicyRules: (_, args, { injector }) =>
+      injector.get(SchemaPolicyApiProvider).listAvailableRules(),
+  },
+  Mutation: {
+    updateSchemaPolicyForOrganization: async (
+      _,
+      { selector, policy, allowOverrides },
+      { injector },
+    ) => {
+      try {
+        const organization = await injector.get(IdTranslator).translateOrganizationId(selector);
+        const config = policyInputToConfigObject(policy);
+        await injector.get(SchemaPolicyApiProvider).validateConfig({ config });
+        const updatedPolicy = await injector
+          .get(SchemaPolicyProvider)
+          .setOrganizationPolicy({ organization }, config, allowOverrides);
+
+        return {
+          ok: {
+            updatedPolicy,
+            organization: await injector.get(OrganizationManager).getOrganization({ organization }),
+          },
+        };
+      } catch (e) {
+        if (e instanceof TRPCClientError) {
+          return formatTRPCErrors(e);
+        }
+
+        return {
+          error: {
+            __typename: 'UpdateSchmeaPolicyResultError',
+            message: (e as Error).message,
+          },
+        };
+      }
+    },
+    updateSchemaPolicyForProject: async (_, { selector, policy }, { injector }) => {
+      try {
+        const translator = injector.get(IdTranslator);
+        const [organization, project] = await Promise.all([
+          translator.translateOrganizationId(selector),
+          translator.translateProjectId(selector),
+        ]);
+        const organizationPolicy = await injector
+          .get(SchemaPolicyProvider)
+          .getOrganizationPolicy({ organization: organization });
+        const allowOverrides = organizationPolicy === null || organizationPolicy.allowOverrides;
+
+        if (!allowOverrides) {
+          throw new Error(
+            `Organization policy does not allow overrides for schema policy at the project level.`,
+          );
+        }
+
+        const config = policyInputToConfigObject(policy);
+        await injector.get(SchemaPolicyApiProvider).validateConfig({ config });
+        const updatedPolicy = await injector
+          .get(SchemaPolicyProvider)
+          .setProjectPolicy({ organization, project }, config);
+
+        return {
+          ok: {
+            updatedPolicy,
+            project: await injector.get(ProjectManager).getProject({ organization, project }),
+          },
+        };
+      } catch (e) {
+        if (e instanceof TRPCClientError) {
+          return formatTRPCErrors(e);
+        }
+
+        return {
+          error: {
+            __typename: 'UpdateSchmeaPolicyResultError',
```

Comment:
```suggestion
            __typename: 'UpdateSchmaPolicyResultError',
```
typo in schema definition?